### PR TITLE
add `SunCalc` definition

### PR DESCRIPTION
**Pull request by horiuchi**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by manuth** [comment]
@horiuchi As you can see in [the source code of dtslint](https://github.com/Microsoft/dtslint/blob/master/src/checks.ts#L60) only a few compiler-options are allowed.

That is:
  - `lib`
  - `noImplicitAny`
  - `noImplicitThis`
  - `strictNullChecks`
  - `strictFunctionTypes`
  - `esModuleInterop`
  - `target`
  - `paths`
  - `jsx`
  - `experimentalDecorators`
  - `noUnusedLocals`
  - `noUnusedParameters`

As you can see `strict` isnt't listed which means this option is not supported by `dtslint`.  
The Travis CI build should work just fine by limiting your `compilerOptions` to the options I've listed.

**Comment by manuth** [comment]
Oh yeah...
I just saw something else in [the source code](https://github.com/Microsoft/dtslint/blob/master/src/checks.ts#L91):
there are, btw, some `compilerOptions` you **must** provide:
  - `noImplicitAny`
  - `noImplicitThis`
  - `strictNullChecks`
  - `strictFunctionTypes`

File: types/suncalc/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strict": true,
```
Comment (manuth) [inline types/suncalc/tsconfig.json]: The Travis CI build will work just fine after you removed this option.

Comment (horiuchi) [reply types/suncalc/tsconfig.json]: Oh, I see.
I removed `strict` option.

**Pull request merged**